platform/saivpp/vpplib/vppxlate/SaiVppXlate.c

```diff
@@ -291,6 +298,23 @@ vl_api_sw_interface_details_t_handler (vl_api_sw_interface_details_t *mp)
     }
 }
 
+static void
+vl_api_create_loopback_instance_reply_t_handler (
+    vl_api_create_loopback_reply_t * msg)
+{
+    vat_main_t *vam = &vat_main;
+
+    set_reply_sw_if_index(ntohl(msg->sw_if_index));
```

Comment:
Setting index in the global structure vat_main not a good idea. The refresh_interfaces function
populates the vpp interface name to sw_if_idx anyway. Why do it here ?

Reply:
Fixed